internal/generateFlatbufferSchema.ts

```diff
@@ -0,0 +1,165 @@
+import { FoxgloveEnumSchema, FoxglovePrimitive, FoxgloveSchema } from "./types";
+
+// Flatbuffers only supports nested vectors via table
+export const BYTE_VECTOR_FB = `
+namespace foxglove;
+
+/// Used for nesting byte vectors
+table ByteVector {
+  data:[uint8];
+}
+root_type ByteVector;
+`;
+
+// Same as protobuf wellknown types
+export const TIME_FB = `
+namespace foxglove;
+
+struct Time {
+  /// 	Represents seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z
+  sec:uint64;
+  /// Nano-second fractions from 0 to 999,999,999 inclusive
+  nsec:uint32;
```

Comment:
I don't think we want uint64+uint32 – the other things I've seen are either a single uint64, or two (u)int32s